pkg/kube/client.go

```diff
@@ -678,6 +680,37 @@ func (c *Client) WaitAndGetCompletedPodPhase(namespace string, reader io.Reader,
 	return status, nil
 }
 
+//GetPodLogs print the log of a pod to standard output
+func (c *Client) GetPodLogs(namespace string, reader io.Reader, timeout time.Duration) error {
+	infos, err := c.Build(namespace, reader)
+	if err != nil {
+		return err
+	}
+	info := infos[0]
```

Comment:
@sorenmat Because the Build method 
```
Build validates for Kubernetes objects and returns resource Infos from a io.Reader.
```
In our case, there will be only one object which will be the first on in the returned objects.

Reply:
👍